### ISSUE TITLE
View of all files in storage

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#11]
## User story
1. As a user
2. I want to be able to see all the files that have been uploaded and are currently stored in the database
3. So that I have an overview of all the files uploaded to the DB.

## Acceptance criteria
* All uploaded files in the database are shown.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
